coldfront/core/utils/templates/upload_mou.html

```diff
@@ -0,0 +1,40 @@
+{% extends "common/base.html" %}
+{% load crispy_forms_tags %}
+{% load humanize %}
+{% load static %}
+{% load feature_flags %}
+
+{% block title %}
+MOU Upload
+{% endblock %}
+
+{% block head %}
+{{ wizard.form.media }}
+{% endblock %}
+
+{% block content %}
+
+<h1>{{ mou_type_long }} MOU Upload</h1>
+<hr>
+
+<script type="text/javascript" src="{% static 'common/js/leave_form_alert.js' %}"></script>
+
+<p>Upload a signed Memorandum of Understanding to be reviewed by our Administrators.</p>
+
+<form method="post" enctype="multipart/form-data">
+  {% csrf_token %}
+  <table>
+  {{ form.as_table }}
+  </table>
+  <br>
+  <input class="btn btn-primary" type="submit" value="Submit" />
+  <a href=
+  {% if mou_type == "new-project" %}"{% url 'new-project-request-detail' request_obj.pk %}"
```

Comment:
It might be easier to just reverse the URL in the view itself, and then pass it via context. Better than hardcoding the types here.